examples/f16/components/fgnetfdm.py

```diff
@@ -103,7 +98,7 @@ class FGNetFDM:
     speedbrake: float = 0
     spoilers: float = 0
 
-    def __init__(self):
+    def __init__(self, parameters: dict = {}):
```

Comment:
This looks like a candidate for switch to keyword arguments for readability.

Reply:
I tried changing this to `def __init__(self, **kwargs: dict = {}):` but I cannot have a default value for `**kwargs` - how do you handle needing an empty default `kwargs` dictionary?

Reply:
I think the idiomatic way in this case is
```python
def __init__(self, 
            elevator_max = FlightGearBase.DEFAULT_FG_ELEVATOR_MAX_DEG,
            ...): ...
```
If there are a lot of parameters, a `Typing.TypedDict` might be a good way of doing this in an explicit (and tool analyzable) way.